### PR TITLE
Mikeechen add classname to dots

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
Deployed to https://beghp.github.io/gh-pages-rc-5<br>It resolves https://github.com/brainhubeu/react-carousel/issues/396<br><br>- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)<br>- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`<br><br>

### CODE REVIEW

**Comment by mikeechen** [comment]
Thank you man! When do you plan on merging this change in?

**Comment by piotr-s-brainhub** [comment]
@mikeechen 

I see `TypeError: this.getProp is not a function` error in some places.
Screenshot 2020-04-02 at 01 51 47
Screenshot 2020-04-02 at 01 51 55
Screenshot 2020-04-02 at 01 51 59
Screenshot 2020-04-02 at 01 52 03

**Comment by mikeechen** [comment]
> @mikeechen
> 
> I see `TypeError: this.getProp is not a function` error in some places.
> Screenshot 2020-04-02 at 01 51 47
> Screenshot 2020-04-02 at 01 51 55
> Screenshot 2020-04-02 at 01 51 59
> Screenshot 2020-04-02 at 01 52 03

My fix is in PR 399

**Comment by piotr-s-brainhub** [comment]
thanks @mikeechen 

we need another review to merge it (I hope it'll happen within the next 12 hours)

**Comment by mikeechen** [comment]
> thanks @mikeechen
> 
> we need another review to merge it (I hope it'll happen within the next 12 hours)

Sounds good! Thank you! Probably need your help on rebasing this branch too.

**Comment by piotr-s-brainhub** [comment]
@mikeechen 

I'll click `Update branch` but before that, a few other PRs could be merged.

File: src/components/CarouselDots.js [hunk 1]
```diff
@@ -67,8 +68,9 @@ class CarouselDots extends Component {
   }
 
   render() {
+    const { className = '', rtl } = this.props;
```
Comment (Lukasz-pluszczewski) [inline src/components/CarouselDots.js:71]: Default value is not needed here.

Comment (mikeechen) [reply src/components/CarouselDots.js:71]: Changed in PR #403

Comment (mikeechen) [reply src/components/CarouselDots.js]: @Lukasz-pluszczewski this is fixed. Please review again :)

**Pull request merged**